tests/numeric.rs

```diff
@@ -336,3 +336,78 @@ fn std_axis_empty_axis()
     assert_eq!(v.shape(), &[2]);
     v.mapv(|x| assert!(x.is_nan()));
 }
+
+#[test]
+#[cfg(feature = "std")]
```

Comment:
I know you copied the tests above, but isn't this the default?  What happends when you remove it?

Reply:
seems to be fine without `#[cfg(feature = "std")]`